.github/workflows/build_and_tests.yml

```diff
@@ -21,7 +27,7 @@ jobs:
       run: sudo apt-get update && sudo apt-get install -y qemu-user-static
     - name: Build & install
       run: |
-        pip install --extra-index-url https://test.pypi.org/simple/ -U .[tests,all_backends]
+        pip install --extra-index-url https://test.pypi.org/simple/ -U .[tests,speculos]
```

Comment:
So `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` should not be superfluous! :)